### DISCUSSION TITLE
How to use with time picker

### DISCUSSION THREAD

**Discussion by marcostaborda** [gpbl/react-day-picker#1474] [Support] [answered]
How to use `react-day-picker` with a time picker?

Does anyone have used `react-day-picker` with time selector?

**Comment by dimitur2204** [comment, marked as answer]
@marcostaborda 

One thing you can do is use the native input [`<input type="datetime-local" />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) which renders an input with date and time input and use that as a way to change time and then use `react-day-picker` to link it to the input https://react-day-picker.js.org/guides/input-fields. 

If you want a time picker as a UI element then you have to do one yourself or just add an extra input field with `type="time"` which has a native time select on most browsers

**Comment by gpbl** [comment]
@dimitur2204 is correct! 

Since this is a FAQ, we would like to add an example with time picker in #1507.

**Reply by dimitur2204** [reply]
That would be an amazing addition to the docs!

**Comment by zisra** [comment]
@dimitur2204 Why not add a ui element time picker?

**Comment by gpbl** [comment]
An example with a time picker integration:

https://react-day-picker.js.org/guides/input-fields#example-time-selection thanks to @Erik-McKelvey
